client/src/components/covid19/USData.js

```diff
@@ -39,10 +39,10 @@ class USData extends React.Component {
                 <YAxis />
                 <Tooltip />
                 <Legend />
-                <Bar dataKey='confirmed' barSize={20} fill='#8884d8' />
-                <Line type='monotone' dataKey='confirmed' stroke='#6562a1' />
-                <Bar dataKey='deaths' barSize={20} fill='#82ca9d' />
-                <Line type='monotone' dataKey='deaths' stroke='#538265' />
+                <Bar dataKey='confirmed' barSize={20} fill='#75D6B1' />
+                <Line type='monotone' dataKey='confirmed' stroke='#75D6B1' />
+                <Bar dataKey='deaths' barSize={20} fill='#757272' />
```

Comment:
can you declare this color as a global variable
e.g.
```js
const primary_color = "#75D6B1 ";
const death_color = "#757272 ";
```

Reply:
color is not fix yet. will fix them and put in global variable  in the next pr